soc/arc/snps_nsim/Kconfig.defconfig.em11d

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2022 Synopsys, Inc. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_NSIM_EM11D
+
+config CPU_EM6
+	default y
+
+config NUM_IRQ_PRIO_LEVELS
+	# This processor supports 4 priority levels:
+	# 0 for Fast Interrupts (FIRQs) and 1-3 for Regular Interrupts (IRQs).
+	default 4
+
+config NUM_IRQS
+	# must be > the highest interrupt number used
+	default 30
+
+config ARC_MPU_VER
+	default 2
+
+config RGF_NUM_BANKS
+	default 2
+
+config SYS_CLOCK_HW_CYCLES_PER_SEC
+	default 5000000
+
+config HARVARD
+	default y
+
+config ARC_FIRQ
+	default y
```

Comment:
Not related to DSP directly - I'm currently reworking firq functionality in Zephyr, so let's not add new simulation platforms with firq without necessity. So, let's have `config ARC_FIRQ -> default n`, `config RGF_NUM_BANKS -> default 1`.

And we need to do corresponding change in `nsim_em11d.props` and `mdb_em11d.args`

Thanks!